scripts/check-version-skew.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function match_with_apiserver_version() {
+  kubelet_version=$(kubectl get "${node_name}" -o jsonpath='{.status.nodeInfo.kubeletVersion}')
+  if [ "${kubelet_version}" != "${apiserver_version}" ]; then
+    echo "${node_name} kubelet version ${kubelet_version}, does not match apiserver."
+    exit 255
```

Comment:
```suggestion
    exit 1
```

Reply:
added `exit 1` with few separators to make sure that this test result looks different from others, but why 1 is appropriate than 255?

Reply:
Well, 255 is a exist code returned by SSH, so it has some special meaning. `exit 1` is general purpose error code, so I think of it as a "standard" exit code for errors, unless you have a reason to change it.

Reply:
TIL, I never had any preference!